src/odin/maneuver.cc

```diff
@@ -166,6 +166,27 @@ bool Maneuver::IsMergeType() const {
           (type_ == DirectionsLeg_Maneuver_Type_kMergeRight));
 }
 
+bool Maneuver::IsLeftType() const {
+  return ((type_ == DirectionsLeg_Maneuver_Type_kUturnLeft ||
```

Comment:
I feel there's some scope of code reuse with whats being done [here](https://github.com/valhalla/valhalla/blob/master/src/odin/maneuversbuilder.cc#L2983). Maybe have the lambda test for `maneuver.IsLeftType() || maneuver. IsRightType()`?

Reply:
Hmmm, or maybe not. I see that some of the types used here are not considered a "turn" in the `is_turn_type` lambda.